tf/aws: remove do not use message (#3559)
buildbuddy-io/buildbuddy: terraform/aws/README.md

# BuildBuddy Example Deployment on AWS with Terraform

> This is **Experimental**, please report any bugs to our Project issue board.

Reviewer: ```suggestion
This is **Experimental**, please report any bugs or submit feedback [here](https://github.com/buildbuddy-io/buildbuddy/issues).
```
